Cargo.toml

```diff
@@ -179,8 +180,11 @@ members = [
   "src/daft-writers",
   "src/hyperloglog",
   "src/parquet2",
-  # "src/spark-connect-script",
-  "src/generated/spark-connect"
+  "src/generated/spark-connect",
+  "src/common/partitioning",
+  "src/daft-py-runners",
+  "src/daft-context",
```

Comment:
Why is there a bunch of stuff being added here that seem unrelated?

Reply:
Faulty merge commit. Removed these additional workspace members.